exchanges/stream/websocket_connection.go

```diff
@@ -326,5 +321,19 @@ func (w *WebsocketConnection) SendMessageReturnResponses(ctx context.Context, si
 		w.Reporter.Latency(w.ExchangeName, outbound, time.Since(start))
 	}
 
+	// Only check context verbosity. If the exchange is verbose, it will log the responses in the ReadMessage() call.
+	if request.IsVerbose(ctx, false) {
+		for i := range resps {
+			log.Debugf(log.WebsocketMgr, "%v %v: Received response [%d/%d]: %v", w.ExchangeName, removeQuery(w.URL), i+1, len(resps), string(resps[i]))
+		}
+	}
+
 	return resps, err
 }
+
+func removeQuery(url string) string {
```

Comment:
```suggestion
func removeURLQueryString(url string) string {
```

or removeURLQuery(url string) / removeURLQueryParams as the current one is a bit too ambiguous

Reply:
🔥